mds/server/settings.py

```diff
@@ -82,3 +90,32 @@
 CORS_ORIGIN_ALLOW_ALL = True
 CORS_ALLOW_CREDENTIALS = True
 CORS_ALLOW_HEADERS = list(default_headers) + ["cache-control"]
+
+REST_FRAMEWORK = {
+    # "DEFAULT_PERMISSION_CLASSES": ("rest_framework.permissions.IsAuthenticated",),
+    "DEFAULT_AUTHENTICATION_CLASSES": (
+        "mds.access_control.stateless_jwt.StatelessJwtAuthentication",
+        "rest_framework.authentication.SessionAuthentication",
+        "rest_framework.authentication.BasicAuthentication",
+    )
+}
+
+AUTH_MEANS = []
+for i in itertools.count(start=1):
+    section = "auth" + (" " + str(i) if i > 1 else "")
```

Comment:
as config may also be an env variable, underscore may be better than space as a separator.

Reply:
indeed